### PR TITLE
BOAC-5640: selectively applies demo mode blur within Vuetify components

### PR DESCRIPTION
**Pull request by lyttam**
https://jira-secure.berkeley.edu/browse/BOAC-5640

### CODE REVIEW

File: src/components/cohort/CohortPageHeader.vue [hunk 1]
```diff
@@ -117,18 +117,18 @@
...
+          <span class="sr-only">Cohort </span>History
         </router-link>
       </div>
-      <div v-if="isCohortHistoryPage" class="d-flex align-self-baseline mr-4">
+      <div v-if="isCohortHistoryPage" class="d-flex align-self-center mr-4">
         <router-link
           v-if="isHistorySupported"
           id="back-to-cohort-link"
-          class="font-size-15 px-1 text-no-wrap"
+          class="v-btn v-btn--variant-text text-anchor text-capitalize font-size-15 px-1 text-no-wrap"
```
Comment (johncrossman) [inline src/components/cohort/CohortPageHeader.vue:131]: @lyttam - Just curious, why is `v-btn` style necessary on `router-link`?  Is this a new standard for all router-link components?

File: src/components/search/AdvancedSearchModal.vue [hunk 2]
```diff
@@ -208,6 +209,7 @@
                       :items="suggestedAdvisors"
                       :maxlength="56"
                       :menu-icon="null"
+                      :menu-props="{'content-class': currentUser.inDemoMode ? 'demo-mode-blur' : ''}"
```
Comment (johncrossman) [inline src/components/search/AdvancedSearchModal.vue:212]: @lyttam - Does this blur the auto-suggestions?

File: src/components/cohort/CohortPageHeader.vue (hunk 1 above)
Comment (lyttam) [reply src/components/cohort/CohortPageHeader.vue:131]: @johncrossman No, I just wanted to make the History link look more like the buttons. Here you can see there are font and alignment differences.

image

File: src/components/search/AdvancedSearchModal.vue (hunk 2 above)
Comment (lyttam) [reply src/components/search/AdvancedSearchModal.vue:212]: Yes.

File: src/components/cohort/CohortPageHeader.vue (hunk 1 above)
Comment (johncrossman) [reply src/components/cohort/CohortPageHeader.vue:131]: @lyttam - Got it. Thanks.

**Pull request merged**